res/css/views/right_panel/_TimelineCard.scss

```diff
@@ -75,6 +75,7 @@ limitations under the License.
 
             .mx_MessageTimestamp {
                 inset-inline: auto 0;
+                font-size: $font-12px;
```

Comment:
see: https://github.com/matrix-org/matrix-react-sdk/commit/78a98415ebfc46ebc5c7d37eb1537eb1756f4f52#diff-f6d1464858812e6ea15c65a49935a089650e5e7adc0a380eeabfce5da896aa67L124 for the original font size.